qiskit/providers/ibmq/ibmqjob.py

```diff
@@ -186,7 +186,21 @@ def result(self, timeout=None, wait=5):
         job_response = self._wait_for_result(timeout=timeout, wait=wait)
         return Result.from_dict(job_response['qObjectResult'])
 
-    def _wait_for_result(self, timeout=None, wait=5):
+    def _get_error_details(self):
```

Comment:
Can you add a docstring - and while you are at it, add it also to `_wait_for_result`, taking advantage of the momentum :grin: ? They are the only two methods missing on the class, and it would be great to have all the class documented.